packages/alfa-style/src/property/text.ts

```diff
@@ -86,6 +88,95 @@ export namespace Text {
     export type Computed = Specified;
   }
 
+  /**
+   * @see https://drafts.csswg.org/css-text/#text-indent-property
+   */
+  export const Indent: Property<
+    Indent.Specified,
+    Indent.Computed
+  > = Property.of(
+    [Length.of(0, "px"), None, None],
+    (input) => {
+      let indent: Length | Percentage | undefined = undefined;
+      let hanging: Option<Indent.Hanging> = None;
+      let eachLine: Option<Indent.EachLine> = None;
+
+      while (true) {
+        for (const [remainder] of Token.parseWhitespace(input)) {
+          input = remainder;
+        }
+
+        if (indent === undefined) {
+          const result = either(Length.parse, Percentage.parse)(input);
+
+          if (result.isOk()) {
+            [input, indent] = result.get();
+            continue;
+          }
+        }
+
+        if (hanging.isNone()) {
+          const result = Keyword.parse("hanging")(input);
+
+          if (result.isOk()) {
+            const [remainder, value] = result.get();
+            hanging = Option.of(value);
+            input = remainder;
+            continue;
+          }
+        }
+
+        if (eachLine.isNone()) {
+          const result = Keyword.parse("each-line")(input);
+
+          if (result.isOk()) {
+            const [remainder, value] = result.get();
+            eachLine = Option.of(value);
+            input = remainder;
+            continue;
+          }
+        }
+
+        break;
+      }
+
+      if (indent === undefined) {
+        return Err.of("Length must be provided");
+      }
+
+      return Result.of([input, [indent, hanging, eachLine]]);
+    },
+    (style) =>
+      style
+        .specified("text-indent")
+        .map(([indent, hanging, eachLine]) => [
+          indent.type === "percentage"
+            ? indent
+            : Resolver.length(indent, style),
+          hanging,
+          eachLine,
+        ]),
+    { inherits: true }
+  );
+
+  export namespace Indent {
+    export type Hanging = Keyword<"hanging">;
+
+    export type EachLine = Keyword<"each-line">;
+
+    export type Specified = [
+      Length | Percentage,
+      Option<Hanging>,
+      Option<EachLine>
+    ];
+
+    export type Computed = [
+      Length<"px"> | Percentage,
+      Option<Hanging>,
+      Option<EachLine>
+    ];
```

Comment:
```suggestion
    export type Specified = Length | Percentage;

    export type Computed = Length<"px"> | Percentage;
```

It turns out that no browsers support `each-line` and `hanging` (https://developer.mozilla.org/en-US/docs/Web/CSS/text-indent#Browser_compatibility) so we won't shouldn't support them either.